### PR TITLE
Stops shipspam

### PR DESCRIPTION
**Pull request by ninjanomnom**
This shouldn't be needed anymore and it's spamming up admin chat

### CODE REVIEW

**Comment by Qustinnus** [comment]
why though :(

**Comment by ninjanomnom** [comment]
Cause it's spammy and unnecesary, it makes it harder to read what people are saying.

**Comment by ike709** [comment]
We could always just have a special log for NPC ship stuff...

File: code/datums/ship.dm [hunk 1]
```diff
@@ -326,7 +326,6 @@ var/next_ship_id
 		ship.attacking_target = chosen_target
 		SSship.broadcast_message("<span class=notice>Caution! [SSship.faction2prefix(ship)] ship ([ship.name]) locking on to [SSship.faction2prefix(ship.attacking_target)] ship ([ship.attacking_target.name]).</span>",null,ship)
 
-	message_admins("[ship.name] has entered into combat at [ship.system]! [ship.attacking_player ? "" : "Combat was not due to players!"]")
```
Comment (monster860) [inline code/datums/ship.dm]: Maybe make it only happen when the combat is triggered by players.

Comment (ninjanomnom) [inline code/datums/ship.dm]: I can do that

Comment (Vivalas) [inline code/datums/ship.dm]: I agree, the earlier behavior was just to help understand what was going on under the hood with ships due to their abstraction, and to help pinpoint erratic behavior with ships if it happens

**Comment by ninjanomnom** [comment]
https://puu.sh/vI3bM/398a4cfc02.png

**Comment by Stealthkibbler** [comment]
Could you maybe change it so the non player triggered ones show up on the news casters under the galactic news channel? I know viv did that with the economy.

**Comment by ninjanomnom** [comment]
That's overscoped for this pr. This is just to make chat more readable

**Pull request merged**